frames/src/main/java/com/checkout/frames/screen/billingformdetails/models/BillingForm.kt

```diff
@@ -0,0 +1,13 @@
+package com.checkout.frames.screen.billingformdetails.models
+
+import com.checkout.base.model.Country
+import com.checkout.tokenization.model.Address
+import com.checkout.tokenization.model.Phone
+import kotlinx.coroutines.flow.MutableStateFlow
+
+internal data class BillingForm(
+    val name: String,
+    val country: MutableStateFlow<Country>,
```

Comment:
We can get rid of mutable state, as we anyway will collect data change for billing address.
And `Address` already has `Country` inside of it, could we use it instead of a separate field?

Reply:
Sure. We can use.